apps/ewallet/lib/ewallet/web/v1/error_handler.ex

```diff
@@ -215,6 +215,10 @@ defmodule EWallet.Web.V1.ErrorHandler do
     exchange_account_wallet_not_found: %{
       code: "exchange:account_wallet_not_found",
       description: "No wallet was found with the given exchange_wallet_address param."
+    },
+    exchange_not_allowed: %{
```

Comment:
Can we rename to `exchange_client_not_allowed`? Thinking there might be other cases that we need to lock down.